enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/SecurityLog.java

```diff
@@ -38,35 +39,31 @@
 import static org.neo4j.helpers.Strings.escape;
 import static org.neo4j.kernel.impl.enterprise.configuration.EnterpriseEditionSettings.security_log_filename;
 
-public class SecurityLog implements Log
+public class SecurityLog implements Log, Lifecycle
```

Comment:
Why not to extend LifecycleAdapter instead for example and just override `shutdown`?